plugins/woocommerce/client/blocks/assets/js/blocks/payment-method-icons/index.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { BlockConfiguration, registerBlockType } from '@wordpress/blocks';
+import { Icon } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import { paymentMethodsIcon } from './icon';
+import edit from './edit';
+import './style.scss';
+
+registerBlockType( metadata.name, {
+	...( metadata as BlockConfiguration ),
+	icon: {
+		src: (
+			<Icon
+				icon={ paymentMethodsIcon }
+				className="wc-block-editor-components-block-icon"
```

Comment:
I don't think this class is used so what about removing it?

```suggestion
```

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/57181/commits/aceeb34fe9fdec47eb175056299ae31b593e3ca3.